BOJ/11047.cpp

```diff
@@ -0,0 +1,32 @@
+#include <bits/stdc++.h>
+using namespace std;
+using ll = long long;
+
+int main() {
+    cin.tie(nullptr)->ios_base::sync_with_stdio(false);
+
+    int N, K;
+    cin >> N >> K;
+
+    vector<int> coins;
+    int tmp;
+    while(N--){
+        cin >> tmp;
+        coins.push_back(tmp);
+    }
```

Comment:
벡터에 N만큼의 공간을 미리 할당해두면 `push_back` 메소드를 사용할때보다 짧고 간결한 코드를 작성할 수 있다
```suggestion
    vector<int> coins(N);
    for(int i=0; i<N; i++) cin >> coins[i];
```